indico/modules/events/papers/templates/emails/revision_submission_to_judge.html

```diff
@@ -0,0 +1,12 @@
+{% extends 'events/papers/emails/revision_submission_base.html' %}
+
+{% block content %}
+    <p>Dear {{ receiver.first_name }},</p>
+    <p>An author has submitted a new revision for the paper <strong>"{{ revision.paper.contribution.title }}"</strong>.</p>
```

Comment:
This is the same as in the reviewer email. Can we use a base class to avoid  repeating it? All that changes is the next paragraph.